source/core/parameters.cc

```diff
@@ -1372,9 +1372,10 @@ namespace Parameters
       prm.declare_entry(
         "type",
         "exponential_decay",
```

Comment:
This is not directly related to this PR, but I was wondering, since we will be using more often the `gaussian_heat_flux_vof_interface` laser `type`, why don't we make it the default?

Additionally, if we do change it, the order of the if/else if/else statements when parsing `(lines 1429-1434)` could also be changed in an order of decreasing likelihood of events (most probable to least). They don't match the ones of the assemblers either in `heat_transfer.cc`.

Reply:
I would not worry about the order of the if, but for sure it can be good to put the default laser to the good one we want to use

Reply:
I will add the factor for the 2D laser in another PR today, and at the same time I'll switch the default laser model :)